src/main/java/com/paneedah/mwc/items/guns/Beowulf50CalFactory.java

```diff
@@ -1044,7 +1044,7 @@ else if(model instanceof AcogReticle) {
                   }, 250, 0))
                   
           .build())
-		.withSpawnEntityDamage(7f)
+		.withSpawnEntityDamage(10f)
```

Comment:
This seems like too much, that's a two shot kill and a one kill headshot (when headshots will be implemented or if they are already)